### PR TITLE
Desktop view

### PR DESCRIPTION
**Pull request by anyepeter**
The entire design for the webpage for the desktop view is completed 👍 

- [x] Responsive to all screen sizes
- [x] Contact form integrated well
- [x] Debugged of markup (HTML)
- [x] Debugged of styling (CSS)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,14 +15,20 @@
...
-                <a class="logo-item" href="index.html">John Doe</a>
+                <a class="logo-item" href="index.html">Peter chendi</a>
 
                 <img class="handberga" src="Images/Icons/Union.png" alt="handberga menu">
-            
+            <ul>
+                <li><a>Portfolio</a></li>
+                <li><a>About</a></li>
+                <li><a>Contact</a></li>
+                <li><a><img src="Images/Icons/Normalbutton/Tertiary/Icons/mail.png" alt="handberga menu"></a></li>
```
Comment (muneebulrehman) [inline index.html]: The hamburger icon should not be visible in the desktop view. Kindly hide this when in desktop mode.
image

File: index.html [hunk 2]
```diff
@@ -31,23 +37,26 @@ <h2>I'am a Software Developer</h2>
...
         </p>
-        
+        </div>
+        <div class="intro-icons">
        <ul>
         <li><img src="Images/Icons/NormalButton/Tertiary/Icons/github-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/Icon-Linkedin.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/hand-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/NormalButton/Tertiary/Icons/twitter-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/medium-icon.png" alt="soclai media icons"></li>
```
Comment (muneebulrehman) [inline index.html]: All the social icons should be links, so visitors can easily find you on other social platforms.

File: index.html [hunk 3]
```diff
@@ -31,23 +37,26 @@ <h2>I'am a Software Developer</h2>
...
        </ul>
-
+        </div>
     </section>
     <section class="recent-work">
         <h2>My recent works</h2> 
 
         <hr>
-        
+        <div class="recent-blog">
```
Comment (muneebulrehman) [inline index.html:55]: This section of your application is not matching the Figma template of your choosing. Kindly follow the exact layout of the Figma design.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: style.css [hunk 4]
```diff
@@ -458,3 +471,568 @@ footer div.icons form > button:focus {
   background-color: #e05304;
   color: #f4f1f0;
 }
+
+@media (min-width: 320px) {
+  .about-container {
```
Comment (muneebulrehman) [inline style.css:476]: Your `about` section is spanning more than 1 vertical screen. Kindly fix it as per the Figma design. Please use the same background colors, padding, margin, font size, etc as in the Figma design. Also, the horizontal line is missing in your design. Kindly include it.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 5]
```diff
@@ -219,7 +229,6 @@ <h2 class="attract">
                       Get in touch
                 </button>
         </form>
-        <hr class="line">
            </div>
 
            <div class="icons">
```
Comment (muneebulrehman) [inline index.html:234]: Again, your app design is not matching the Figma template. Kindly update this section as per the given design. Also, include the horizontal line as in the original design.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @anyepeter , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (levy002) [reply style.css:476]: - [ ] Kindly make sure to implement the suggested changes by adding the correct background color to the `about` section.

File: style.css [hunk 6]
```diff
@@ -458,3 +471,623 @@ footer div.icons form > button:focus {
...
+  section.recent-work {
+    margin-bottom: 0%;
+  }
+
+  section.about-container div {
+    background-image: url("/Images/abvout.png");
+    background-position: 533px;
+    background-size: 697px;
+    background-position-y: 257px;
+  }
```
Comment (levy002) [inline style.css]: - [x] As requested by the previous reviewer, kindly change `background-color` of the `about` section to `#262626` to look like the provided design. :100: 

  - ### Your design
 
   image

   - ### Figma design
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @anyepeter,  :wave: 
### I'm Levy, your PR reviewer

Great job so far :+1:  
However, they are some issues that you need to address before you move on to the next project :wink:

## Highlights
- Good job for using the correct Github flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
### Status: REQUESTED CHANGES :recycle: 
_check my comments under review_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

 - [OPTIONAL] Kindly make sure to add a good PR description. It makes your repo looks more professional. :wink:  Kindly go ahead and edit your PR title and description.

   - Example of good PR
   
     PR Title: Hello Microverse
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 6 above)
Comment (anyepeter) [reply style.css]: Thanks for the review

**Review by mwanawabangona — APPROVED** [review]
Hi @anyepeter  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**